packages/store/src/workspace/database.ts

```diff
@@ -21,12 +21,16 @@ export class DatabaseManager {
   }
 
   protected get yCells() {
```

Comment:
The migration to block level props looks great, while seems that we are still actually storing the database props on the root page block? We can move this in following PRs.